spec.md

```diff
@@ -445,9 +445,99 @@ Resource properties:
 * `name` - optional
 * Links to any metadata objects controlled by the DataPub server
 * Links to any metadata objects modifiable by the resource's controller
-### Resource Operations
 
-#### Resource HTTP API
+### Create Resource Operation
```

Comment:
I think we should:
* only consider this operation recommended for ease of use, not production, because it's not idempotent
* allow servers to diable it in prod (i.e. supporting the operation is not a requirement)
* defined in terms of the equivalent PUT
* put this below 'Update'/PUT